_data/sites.json

```diff
@@ -4266,13 +4266,14 @@
         "meta": "popular",
         "name": "Netflix",
         "url": "https://help.netflix.com/en/node/407",
-        "difficulty": "impossible",
-        "notes": "Contact customer services. Even then they may not delete your account under the premise that you might want to rejoin and keep your history and recommendations.",
-        "notes_fr": "Contactez le service clients.",
+        "difficulty": "hard",
+        "notes": "Cancel your membership, then contact support if you want immediate deletion. Otherwise your account will be conserved for 10 months.",
```

Comment:
```suggestion
        "notes": "Cancel your membership, then contact support if you want immediate deletion, otherwise your account will be automatically deleted in 10 months.",
```